packages/core/components/InputOrGroup/styles.module.css

```diff
@@ -48,7 +48,8 @@
   border-color: var(--puck-color-grey-8);
   border-radius: 4px;
   font-family: inherit;
-  padding: 12px 16px;
+  font-size: 16px;
+  padding: 10.25px 15px;
```

Comment:
Where's this fractional value coming from?

Reply:
To match the existing height with the new `font-size`